autonomi/python/examples/autonomi_scratchpad.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+"""
+Autonomi Scratchpad Example
+
+This example demonstrates how to use the scratchpad functionality in Autonomi,
+including how the counter is automatically managed during updates.
+
+Scratchpads are mutable, encrypted data containers on the Autonomi network.
+Each scratchpad is tied to a specific public key and can be updated by the owner.
+"""
+
+from autonomi_client import (
+    Client, SecretKey, Wallet, PaymentOption, Network, Bytes,
+    Scratchpad, ScratchpadAddress
+)
+import asyncio
+import time
+
+async def main():
+    # Initialize client
+    print("Connecting to Autonomi network...")
+    client = await Client.init_local()  # For testnet use Client.init()
+    
+    # Create wallet for payment
+    print("Setting up wallet...")
+    network = Network(True)  # Testnet
+    private_key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"
+    wallet = Wallet.new_from_private_key(network, private_key)
+    payment = PaymentOption.wallet(wallet)
+    
+    # Create secret key for the scratchpad (this represents your identity)
+    key = SecretKey.random()
```

Comment:
The (empty) constructor generates a random key. The `random` method does not exist.
```suggestion
    key = SecretKey()
```